enterprise/views.py

```diff
@@ -1704,11 +1709,17 @@ def post(self, request, enterprise_uuid, course_id):
                 data_sharing_consent
             )
 
-        user_consent_needed = get_data_sharing_consent(
-            enterprise_customer_user.username,
-            enterprise_customer.uuid,
-            course_id=course_id
-        ).consent_required()
+        # If enterprise-catalog is not enabled it is necessary disable data sharing consent
+        # for enterprise customers.
+        user_consent_needed =  False
+
+        if not configuration_helpers.get_value('DISABLE_DATA_SHARING_CONSENT', False):
+            user_consent_needed = get_data_sharing_consent(
+                enterprise_customer_user.username,
+                enterprise_customer.uuid,
+                course_id=course_id,
+            ).consent_required()
```

Comment:
Could you please try this inline if?, this way we don't indent anything. @anfbermudezme 

```suggestion
        user_consent_needed = get_data_sharing_consent(
            enterprise_customer_user.username,
            enterprise_customer.uuid,
            course_id=course_id,
        ).consent_required() if not configuration_helpers.get_value(
            'DISABLE_DATA_SHARING_CONSENT',
            False,
        ) else False
```